Services/CO.CDP.DataSharing.WebApi/Model/Form.cs

```diff
@@ -36,8 +51,8 @@ public record Form
     public required string ShareCode { get; init; }
 
     /// <example>0</example>
-    public required int Scope { get; init; }
+    public required FormScope Scope { get; init; }
 
     /// <example>0</example>
-    public required int Type { get; init; }
+    public required FormSectionType Type { get; init; }
```

Comment:
This should be defined on the section rather than form, since we moved it there.

Reply:
there is no FormSection Model in the DataSharing API.